lib/travis/api/v3/services/enterprise_license/find.rb

```diff
@@ -31,12 +23,11 @@ def run!
     private
 
     def replicated_endpoint
-      ENV['REPLICATED_INTEGRATIONAPI']
+      ENV['REPLICATED_LICENSELICENSEID']
```

Comment:
do we need replicated_endpoint function at all? how is it related to license_id?